healthcheck/core/bnd.bnd

```diff
@@ -1,17 +1,8 @@
-Bundle-Category: healthcheck
-
-Bundle-Description: ${project.description}
-
-Bundle-DocURL: https://felix.apache.org
-
-Bundle-License: Apache License, Version 2.0
-
-Bundle-Vendor: The Apache Software Foundation
-
-Import-Package: org.quartz*;resolution:="optional", *
-
-Conditional-Package: org.apache.felix.utils.*
-
--removeheaders:\
-  Include-Resource,\
-  Private-Package
+Bundle-Category       : healthcheck
+Bundle-Description    : ${project.description}
+Bundle-DocURL         : https://felix.apache.org
+Bundle-License        : Apache License, Version 2.0
+Bundle-Vendor         : The Apache Software Foundation
+Import-Package        : org.quartz*;resolution:="optional", *
+-conditionalpackage   : org.apache.felix.utils.*, org.apache.commons.lang3.*
```

Comment:
Checked a build with and without conditional package - it looks like the whole commons lang library is pulled in ATM and hence the size of the hc core bundle is just the regular jar + the 435KB from commons lang ending up at a similar memory footprint. @amitjoy Can you do some tests on how to really get the file size down? (the optional include only taking what's needed works for org.apache.felix.utils.*, not sure why it does not for org.apache.commons.lang3.*)

Reply:
I have also tested it yesterday and have noticed this discrepancy. The internal references to other different packages lead to the inclusion of several packages of org.apache.commons.lang3 which results to extra ~250KB in respective bundles. For the time being, I would propose to leave it as it is now. I will, in the meanwhile, think about it how we can trim down the references. We can definitely reimplement the referenced static factories from Apache Commons Lang3 once again that I tried too. But I feel it's rather reinventing the wheel. I will think about some slim-down approaches and keep you updated.